locales/en-US/messages.properties

```diff
@@ -88,19 +88,3 @@ i_donated_to_thunderbird=I donated to @mozthunderbird today to #freetheinbox. Jo
 
 # Site title when shared on Facebook
 support_mozilla=Support Mozilla
-
-
-# Legacy strings
-first_name=First name
-last_name=Last name
-address=Address
-city=City
-country=Country
-postal_code=Zip/Postal code
-state_province=State/Province
-
-# donate-button.js
-donate_now_amount=Donate { donationAmount } now
```

Comment:
This is used here:

![donate_now_amount](https://cloud.githubusercontent.com/assets/197334/20316724/0d5117f2-ab31-11e6-80f2-be721c5b519f.png)

Just select an amount after using a non paypal currency. It's also used on our https://donate.mozilla.org/en-US/paypal-donate page, but that might be a little less critical, as I've wanted to redo that page for a bit now.